constants.py

```diff
@@ -1 +1,2 @@
-
+# Elevator
+elevator_length_per_rotation = 0.1 # Change later, this isn't accurate
```

Comment:
the current ratio is 0.0225 meters per revolution, so you can use that ratio to determine the height